src/http/server.ts

```diff
@@ -331,10 +332,21 @@ export class Server {
         if (isNaN(e.code)) {
           e.code = 500;
         }
-        return new Response(e.stack, {
-          status: e.code,
-          headers: response.headers,
-        });
+
+        if (exception) {
+          try {
+            await exception.catch(e, originalRequest, response);
```

Comment:
this isn't documented in the codebase, but we try to instantiate things during compile time so that

1. the user doesn't have to do it
2. saves the server from having to build a new object for every request

could you make it so that the exception doesn't have to be instantiated by the user? that's the only feedback i have. id like to get thoughts on the naming of the class from @ebebbington and @Guergeiro

as far as implementation, LGTM. thanks @routmoute !

Reply:
You mean the `new Drash.ExceptionLayer().catch(...)`?

Reply:
yea, but we instantiate it once and reuse it just like we do with resources

Reply:
I agree with it. Let's avoid generating garbage for the garbage collector to collect 👍🏿

Reply:
ok, last commit instantiate ExceptionLayer on server.ts